api/schema/tag_to_label_resource.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Illumio, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var (
+	tagToLabelResource = Resource{
+		TypeName: "tag_to_label",
+		Schema: resource_schema.Schema{
+			Version:             1,
+			MarkdownDescription: "Maps AWS account tags to CloudSecure labels.",
+			Attributes: map[string]resource_schema.Attribute{
+				IDFieldName: idAttribute,
+				"cloud_tags": resource_schema.ListAttribute{
+					MarkdownDescription: "List of tags to map to CloudSecure labels with the specified key. The values of the created labels correspond to the values of the tags. The cloud field for each tag must be \"aws\" or \"azure\" or \"oci\".",
```

Comment:
Don't refer to OCI, until we properly support onboarding OCI accounts in this Terraform provider.